React/Modules/RCTRedBox.m

```diff
@@ -169,6 +169,14 @@ - (void)copyStack
   
   UIPasteboard *pb = [UIPasteboard generalPasteboard];
   [pb setString:fullStackTrace];
+  
+#if TARGET_IPHONE_SIMULATOR
+  // if running in the simulator, try to get the stack up to the hosting computer's clipboard as
+  // well as the pastboard in the simulator
+  if (_actionDelegate && [_actionDelegate respondsToSelector:@selector(redBoxWindow:sendTextToPasteboard:)]) {
+    [_actionDelegate redBoxWindow:self sendTextToPasteboard:fullStackTrace];
+  }
+#endif
```

Comment:
The whole logic in this `copyStack` method should be moved to the actionDelegate passing the necessary data.
I'll let @javache confirm this.